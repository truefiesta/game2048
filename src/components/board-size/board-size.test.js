import React from "react";
import renderer from "react-test-renderer";
import BoardSize from "./board-size";

describe(`BoardSize snapshot`, () => {
    it(`renders`, () => {
        const boardSize = renderer.create(
            <BoardSize
                size={4}
                onSelect={() => null}
            />
        )
        .toJSON();

        expect(boardSize).toMatchSnapshot();
    });
});
