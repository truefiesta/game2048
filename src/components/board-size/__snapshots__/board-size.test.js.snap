// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BoardSize snapshot renders 1`] = `
<div
  className="board-size"
>
  <label
    className="subsection-title"
    htmlFor="size-select"
  >
    Board size: 
  </label>
  <select
    className="size-select"
    id="size-select"
    name="size"
    onChange={[Function]}
    value={4}
  >
    <option
      value="4"
    >
      4 * 4
    </option>
    <option
      value="8"
    >
      8 * 8
    </option>
  </select>
</div>
`;
